proposals/0064-remote-control.md

```diff
@@ -0,0 +1,822 @@
+    </param>
+  </function>
+  <function name="GetInteriorVehicleDataConsent" messagetype="response">
+    <param name="allowed" type="Boolean" mandatory="true">
+      <description>"true" - if the driver grants the permission for controlling to the named app; "false" - in case the driver denies the permission for controlling to the named app.</description>
+    </param>
+  </function>
+
+  <function name="OnInteriorVehicleData" functionID="OnInteriorVehicleDataID" messagetype="notification">
+    <param name="moduleData" type="Common.ModuleData">
+    </param>
+  </function>
+  <function name="OnReverseAppsAllowing" messagetype="notification">
+    <description>Sender: vehicle -> RSDL. Notification about remote-control for passenger`s applications must be either turned off or turned on. Sent after User`s choice through HMI.</description>
+    <param name="allowed" type="Boolean" mandatory="true" >
+      <description>If "true" - RC for passengers is allowed; if "false" - disallowed.</description>
+    </param>
+  </function>
+  <function name="OnDeviceRankChanged" messagetype="notification">
+    <description>Sender: vehicle->RSDL. Purpose: inform about the device is set either as driver`s or as passenger`s  device by the HMI settings or by the HMI User through the appropriate menu.</description>
+    <param name="deviceRank" type="Common.DeviceRank" mandatory="true">
+      <description>If "DRIVER" - the named by "DeviceInfo" device is set as driver`s. If "PASSENGER" -  the named by "DeviceInfo" device is set as passenger`s. </description>
+    </param>
+    <param name="device" type="Common.DeviceInfo" mandatory="true">
+      <description>The device info: name and ID. Initially is sent by SDL via UpdateDeviceList to the vehicle.</description>
+    </param>
+  </function>
+  <function name="OnDeviceLocationChanged" messagetype="notification">
+    <description>Sender: vehicle->RSDL. Purpose: inform about the device location in the vehicle interior.</description>
+    <param name="deviceLocation" type="Common.InteriorZone" mandatory="true">
+      <description>Defines the vehicle zone where the named device is located. </description>
+    </param>
+    <param name="device" type="Common.DeviceInfo" mandatory="true">
+      <description>The device info: name and ID. Initially is sent by SDL via UpdateDeviceList to HMI.</description>
+    </param>
+  </function>
+</interface>
+```
+
+## Potential downsides
+
+- The driver distraction rules and regional regulations are getting more stringent on driver's using mobile phones while driving. Current design cannot prevent a driver mark a phone as PASSENGER and use it while driving.
+
+- The driver must exit current control application before using another application to control the same RC module. There is no indication of which application is currently control the RC module. Driver doesn't know which application to be closed. This makes application switching cumbersome.
```

Comment:
Please change "control" to "controlling" and "application to be closed" to "application to close"